projects/demo/src/modules/app/classes/ts-file.parser.ts

```diff
@@ -0,0 +1,49 @@
+const CLASS_NAME_REGEXP = /(?<=export class\s)(\w*)/gi;
+
+export class TsFileParser {
+    get className(): string {
```

Comment:
can there be multiple exports in the file?

Reply:
it can be.
but we should not do it in the examples. It too complex too process.

Reply:
1 component/module per file

Reply:
`but we should not do it in the examples. It too complex to process.` so, you can throw exception for it when file have more export references?

Reply:
done